satellite/console/consoleweb/server.go

```diff
@@ -117,8 +125,42 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, mail
 	return &server
 }
 
+// Run starts the server that host webapp and api endpoint
+func (server *Server) Run(ctx context.Context) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	server.schema, err = consoleql.CreateSchema(server.log, server.service, server.mailService)
+	if err != nil {
+		return Error.Wrap(err)
+	}
+
+	err = server.initializeTemplates()
+	if err != nil {
+		// TODO: should it return error if some template can not be initialized or just log about it?
```

Comment:
Yes, failing to start-up is usually a safer approach.

Reply:
but in this case tests will fail, because they didn't generate dist folder

Reply:
The templates aren't generated so there should be a template path they can use.